docs/docsite/.templates/banner.html

```diff
@@ -50,6 +50,8 @@
         banner += extra_banner;
 
         msg += 'You are reading the <b>devel</b> version of the Ansible documentation - this version is not guaranteed stable. Use the version selection to the left if you want the <b>latest</b> (stable) released version.';
+        /* temp - add extra warning about core-2.19. This should move to latest at release time */
+        msg += '<br><p><strong>Important: </strong>The ansible-core 2.19/Ansible 12 release has made significant templating changes that enable reporting of numerous problematic behaviors that went undetected in previous releases, with wide-ranging positive effects on security, performance, and user experience. Use the <a href="https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_12.html">porting guide</a> to understand where you may need to update your playbooks and roles.';
```

Comment:
Here's how I would write this - the suggested changes slim the warning from 50 words / 442 characters (with spaces) to 44 words / 308 characters (with spaces):
```suggestion
        msg += '<br><p><strong>Important: </strong>The ansible-core 2.19/Ansible 12 release has made significant changes to templating. These changes improve security, performance, and user experience by reporting problematic behaviors that were allowed in previous releases. You may need to update your playbooks and roles. See the <a href="https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_12.html">porting guide</a> for details.';
```